usersession/agent/rest_api_test.go

```diff
@@ -935,84 +947,107 @@ func addAppToSnap(c *C, snapinfo *snap.Info, app string, isService bool, icon st
 		newInfo.Daemon = "daemon"
 	}
 	snapinfo.Apps[app] = &newInfo
-	createDesktopFile(c, &newInfo, icon)
+	createDesktopFile(c, newInfo.DesktopFile(), icon, name, nil)
 }
 
 func (s *restSuite) TestGuessAppIconNoIconPrefixEqualApp(c *C) {
```

Comment:
Tiny nitpick, I think all `TestGuessAppIcon*` test can be renamed to `TestGuessAppData*`.

Reply:
Mmmm... good point.

Reply:
Done.